Initium-Odp/src/com/universeprojects/miniup/server/scripting/wrappers/EntityWrapper.java

```diff
@@ -201,6 +201,16 @@ public boolean setHardcoreMode(boolean hcmMode)
 		return false;
 	}
 	
+	/**
+	 * JS numbers don't translate cleanly; here, we can make sure that we are actually passing a LONG.
+	 * Useful for changing DR, int mod, dex mod, str mod, etc.
+	 * @param property
+	 * @param value
+	 */
+	public void setLongProperty(String property, Long value) {
+		this.setProperty(property, value);
+	}
+	

```

Comment:
This isn't allowed. Certain fields we _don't_ allow changing through the wrapper objects. This would basically expose every single property to scripting environment, which we try to expose only through certain fields/methods.